conf/base/catalog.yml

```diff
@@ -0,0 +1,29 @@
+works_for_concepts:
+  type: ${_json.type}
+  filepath: s3://alphafold-impact/data/01_raw/openalex/works/works_for_concepts.json
+  credentials: s3_credentials
+
+works_incoming_citations:
+  type: partitions.PartitionedDataset
+  dataset: ${_json.type}
+  filename_suffix: ".json"
+  path: s3://alphafold-impact/data/02_intermediate/openalex/works/cited_by/
+  credentials: s3_credentials
+
+works_outgoing_citations:
+  type: partitions.PartitionedDataset
+  dataset: ${_json.type}
+  filename_suffix: ".json"
+  path: s3://alphafold-impact/data/02_intermediate/openalex/works/cites/
+  credentials: s3_credentials
+
+downstream_incoming_citations:
+  type: partitions.PartitionedDataset
+  dataset: ${_json.type}
+  filename_suffix: ".json"
+  path: "s3://alphafold-impact/data/02_intermediate/openalex/works/cited_by/downstream/${globals:get.work_id, undefined}/"
+  credentials: s3_credentials
+
+# TODO: The intention is for downstream_incoming_citations to create a subfolder given the work_id. This
+# is not working at the moment, although it is possible using runtime arguments.
+# https://docs.kedro.org/en/stable/configuration/advanced_configuration.html#how-to-override-configuration-with-runtime-parameters-with-the-omegaconfigloader
```

Comment:
Is this functionality working?

Reply:
Yes! Old comments that I forgot to remove, good catch!

Reply:
You can see how OmegaConf is interpolating variables between globals vars (it uses a default resolver for it, you can also pass [custom resolvers](https://docs.kedro.org/en/latest/kedro.config.OmegaConfigLoader.html)!) and the placeholder in catalog.yml. If it doesn't find any, it uses "undefined" as default folder name.

Reply:
You can also see how that's an issue.. in `settings.py`, I've made it so that OmegaConf sees all parameters as global vars. Not bueno, but works for now...